README.md

```diff
@@ -24,7 +24,7 @@ Python module for reading and writing UNTL metadata records.
 Requirements
 -------------
 
-* Python ~= 2.7
+* Python ~= 3.7
```

Comment:
We have tests passing on 3.6 and 3.7, so maybe we should change this to something like Python 3.6-3.7 or something that reflects more than 3.7.x.